sdk/search/azure-search-documents/tests/async_tests/test_service_live_async.py

```diff
@@ -45,3 +57,133 @@ class SearchIndexClientTest(AzureMgmtTestCase):
         result = await client.get_service_statistics()
         assert isinstance(result, dict)
         assert set(result.keys()) == {"counters", "limits"}
+
+    @ResourceGroupPreparer(random_name_enabled=True)
+    @SearchServicePreparer()
+    @await_prepared_test
```

Comment:
That should not be necessary now, @KieranBrantnerMagee made an update for this to work automatically